pkg/iac/scanners/terraform/parser/resolvers/registry.go

```diff
@@ -59,17 +60,20 @@ func (r *registryResolver) Resolve(ctx context.Context, target fs.FS, opt Option
 
 		token, err = getPrivateRegistryTokenFromEnvVars(hostname)
 		if err == nil {
-			opt.Debug("Found a token for the registry at %s", hostname)
+			opt.DebugLogger.Debug("Found a token for the registry", log.String("hostname", hostname))
```

Comment:
It seems a little too verbose to have `opt.DebugLogger.Debug`. Could we rename it better?

Reply:
Renamed `DebugLogger` to `Logger` https://github.com/aquasecurity/trivy/pull/7295/commits/b2cdb991691a5f544bcda9232eced6b2385c0f24